src/rtc/h5_prep.py

```diff
@@ -667,39 +701,98 @@ def get_metadata_dict(product_id: str,
-             STANDARD_RTC_S1_ONLY,
+             'Flag to indicate if radiometric terrain correction (RTC) has'
+             ' been applied'],
+        'metadata/processingInformation/parameters/' +
+        'dryTroposphericGeolocationCorrectionApplied':
+            ['processing_information'
+             '_dry_tropospheric_geolocation_correction_applied',
+             ALL_PRODUCTS,
              cfg_in.groups.processing.apply_dry_tropospheric_delay_correction,
-             'Flag to indicate if the dry tropospheric correction has been applied'],
-        'metadata/processingInformation/parameters/wetTroposphericGeolocationCorrectionApplied':
-            ['wet_tropospheric_geolocation_correction_applied',
-             STANDARD_RTC_S1_ONLY,
+             'Flag to indicate if the dry tropospheric correction has been'
+             ' applied'],
+        'metadata/processingInformation/parameters/' +
+        'wetTroposphericGeolocationCorrectionApplied':
+            ['processing_information'
+             '_wet_tropospheric_geolocation_correction_applied',
+             ALL_PRODUCTS,
              False,
-             'Flag to indicate if the wet tropospheric correction has been applied'],
-        'metadata/processingInformation/parameters/bistaticDelayCorrectionApplied':
-            ['bistatic_delay_correction_applied',
-             STANDARD_RTC_S1_ONLY,
+             'Flag to indicate if the wet tropospheric correction has been'
+             ' applied'],
+        'metadata/processingInformation/parameters/' +
+        'bistaticDelayCorrectionApplied':
+            ['processing_information'
+             '_bistatic_delay_correction_applied',
+             ALL_PRODUCTS,
              cfg_in.groups.processing.apply_bistatic_delay_correction,
-             'Flag to indicate if the bistatic delay correction has been applied'],
+             'Flag to indicate if the bistatic delay correction has been'
+             ' applied'],
+        'metadata/processingInformation/parameters/' +
+        'inputBackscatterNormalizationConvention':
+            ['processing_information'
+             '_input_backscatter_normalization_convention',
+             ALL_PRODUCTS,
+             cfg_in.groups.processing.rtc.input_terrain_radiometry,
+             'Backscatter normalization convention of the source data'],
+        'metadata/processingInformation/parameters/' +
+        'outputBackscatterNormalizationConvention':
+            ['processing_information'
+             '_output_backscatter_normalization_convention',
+             ALL_PRODUCTS,
+             cfg_in.groups.processing.rtc.output_type,
+             'Backscatter normalization convention of this product (RTC-S1)'],
+        'metadata/processingInformation/parameters/' +
+        'outputBackscatterExpressionConvention':
+            ['processing_information'
+             '_output_backscatter_expression_convention',
+             ALL_PRODUCTS,
+             'linear power/intensity',
```

Comment:
I think it has to be either `linear power` or `linear intensity`

Reply:
I think backscatter (coefficient) is a measurement of intensity, i.e., power per area, but one can also consider it the power received for a give radar bin. Usually, power and intensity are used interchangeably meaning that you take the square of the amplitude (volts). In CARD specifications they list two options: "linear amplitude" or "linear power", but since I prefer "intensity", I put "linear power/intensity". Maybe I can just go ahead and use "intensity". Any thoughts? @hfattahi , if you have any preference, please let us know.

Reply:
Thanks for the explanation. I think the point of the CARD 3.2 (scaling conversion) is to clarify if the values are squared or not. To my understanding, both "intensity" and "power" are based on the square of "amplitude", so I thought  `power/intensity` are duplicating.

Reply:
Just updated it to "linear backscatter intensity"